client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,351 @@
+							onChange={ this.handleChangeEvent } />
+						<span>{ translate( 'A company or organization' ) }</span>
+					</FormLabel>
+				</FormFieldset>
+
+				{ 'individual' === registrantType
+					? this.renderPersonalFields()
+					: this.renderOrganizationFields() }
+
+				{ this.props.children }
+			</form>
+		);
+	}
+
+	renderPersonalFields = () => {
+		const translate = this.props.translate;
+		const {
+			countryOfBirth,
+			dob_days,
+			dob_months,
+			dob_years,
+			placeOfBirth,
+			postalCodeOfBirth,
+		} = { ...emptyValues, ...this.props.values };
+
+		return (
+			<div>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Country of Birth' ) }
+					</FormLabel>
+					<FormCountrySelect
+						name="countryOfBirth"
+						value={ countryOfBirth }
+						countriesList={ this.props.countriesList }
+						onChange={ this.handleChangeEvent } />
+				</FormFieldset>
+
+				<FormFieldset>
+					<FormLegend>
+						{ translate( 'Date of Birth' ) }
+					</FormLegend>
+					<div className="registrant-extra-info__dob-inputs">
+						<div className="registrant-extra-info__dob-column">
+							<FormLabel>
+								{ translate( 'Year' ) }
+							</FormLabel>
+							<FormTextInput className="registrant-extra-info__dob-year"
+								name="dob_years"
+								value={ dob_years }
+								type="number"
+								placeholder="YYYY"
+								onChange={ this.handleDobChangeEvent } />
+						</div>
+						<div className="registrant-extra-info__dob-column">
+							<FormLabel>
+								{ translate( 'Month' ) }
+							</FormLabel>
+							<FormTextInput className="registrant-extra-info__dob-month"
+								name="dob_months"
+								value={ dob_months }
+								type="number"
+								placeholder="MM"
+								onChange={ this.handleDobChangeEvent } />
+						</div>
+						<div className="registrant-extra-info__dob-column">
+							<FormLabel>
+								{ translate( 'Day' ) }
+							</FormLabel>
+							<FormTextInput className="registrant-extra-info__dob-day"
+								name="dob_days"
+								value={ dob_days }
+								type="number"
+								placeholder="DD"
+								onChange={ this.handleDobChangeEvent } />
+						</div>
+					</div>
+					<FormSettingExplanation>{
+						translate( 'e.g. 1970 12 31', {
+							comment: 'This is describing a date format with fixed fields, so please do not ' +
+								'alter the numbers (Year, Month, Day). Please translate e.g("For example") if appropriate.'
+						} )
+					}</FormSettingExplanation>
+				</FormFieldset>
+
+				{ countryOfBirth === 'FR' && (
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Place of Birth' ) }
+						</FormLabel>
+						<FormTextInput
+							name="placeOfBirth"
+							value={ placeOfBirth }
+							placeholder={ translate( 'Place or city of birth' ) }
+							onChange={ this.handleChangeEvent } />
+					</FormFieldset>
+				) }
+
+				{ countryOfBirth === 'FR' && (
+					<FormFieldset>
+						<FormLabel>
+							{ translate( 'Postal Code of Birth' ) }
```

Comment:
I know, right!?  On the upside, AFNIC has dropped this, let's just hope OpenSRS does too.